src/Nova/Phone.php

```diff
@@ -14,7 +14,7 @@ class Phone extends BaseResource
 {
     public static $model = \Tipoff\Addresses\Models\Phone::class;
 
-    public static $title = 'id';
+    public static $title = 'full_number';
```

Comment:
Let's use a formatted number accessor for this on the model. Example:

407-555-5555

If the number is not in the US, then the accessor should include the country calling code:

+506 2696 0000

Reply:
Let's also create another accessor on the model for parenthetical formatted numbers. This would return the following:

(407) 555-5555

Sometimes we may use that format on the website or in an email, invoice, etc.

Reply:
Hey @drewroberts done with the formatted number accessor and the parenthetical formatted numbers... however; with the international format is complex, because it depends on the country... after read a lot about it

E.G

+593984261408 (Ecuador) => translate into this    +593 98 426 1408
+584147587761 (Venezuela) => translate into this +58 4147587761
+56949313153 (Chile) => translate into this           +56 9 4931 3153

Reply:
We have a formatted style of those international country calling codes in that table. Just use that and don't worry about formatting the rest of the numbers for now. We'll figure out that later if it becomes necessary.